tools/genaudit/genaudit/templates/macros.rst

```diff
@@ -1,6 +1,6 @@
 {% macro short_author(author) %}`@{{ author.nick }} <{{ author.url }}>`_{% endmacro %}
 {% macro short_authors_list(authors) %}{% for author in authors %}{{ short_author(author) }}{{ ", " if not loop.last else "" }}{% endfor %}{% endmacro %}
-{% macro approvers_and_auditer_list(auditer, approvers) %}{% if approvers or auditer %}{{ short_authors_list(approvers) }}{% if approvers and auditer %},{% endif %}{% if auditer %}({{ short_author(auditer) }}){% endif %}{% else %}none{% endif %}{% endmacro %}
+{% macro approvers_and_auditer_list(auditer, approvers) %}{% if approvers or auditer %}{{ short_authors_list(approvers) }}{% if approvers and auditer %}, {% endif %}{% if auditer %}({{ short_author(auditer) }}){% endif %}{% else %}none{% endif %}{% endmacro %}
```

Comment:
Missing whitespace after comma between the list of GitHub approvers and the BSI project auditor (in parenthesis). This lead to funny line breaks and overflowing content boxes:

![image](https://github.com/sehlen-bsi/botan-docs/assets/1562139/98ce9dad-9d17-493a-ab95-9f64cb1bdc29)